src/components/ContentSidebar/SidebarNotices.js

```diff
@@ -4,49 +4,73 @@
  * @author Box
  */
 
-import React from 'react';
+import React, { Fragment } from 'react';
 import getProp from 'lodash/get';
 import SharedLinkExpirationNotice from 'box-react-ui/lib/features/item-details/SharedLinkExpirationNotice';
-import SidebarSection from './SidebarSection';
+import ItemExpirationNotice from 'box-react-ui/lib/features/item-details/ItemExpirationNotice';
 import { addTime } from '../../util/datetime';
 import type { BoxItem } from '../../flowTypes';
 import DateField from '../Date';
 
 const ONE_MINUTE_IN_MS = 60000;
 
+const NOTICE_DATE_FORMAT = {
+    month: 'short',
+    day: 'numeric',
+    year: 'numeric'
+};
+
 type Props = {
     file: BoxItem
 };
 
+/**
+ * Adds one minute to a ISO formatted date string
+ *
+ * @private
+ * @param {string} expiration - an item or shared link expiration
+ * @return {string} an ISO formatted date string
+ */
+export function addMinuteToExpiration(expiration: string) {
+    let expirationDate = new Date(expiration);
+    // One minute is added to account for dates set via a date picker.
+    // These dates will actually be stored as 11:59PM the night before the item expires.
+    expirationDate = addTime(expirationDate, ONE_MINUTE_IN_MS);
```

Comment:
Any reason we need to new Date() above, as well as do .toISOString()?
Datefield should be localization formatting.

Does adding this inline, make the code more un-readable? Because it can then remove the if() blocks.

```js
<DateField date={addTime(expirationDate, ONE_MINUTE_IN_MS)} dateFormat={NOTICE_DATE_FORMAT} relative={false} />
```

Reply:
addTime takes in either a date or a timestamp. The API returns an ISO formatted string, so we need to at least convert it to a Date. I can remove the toISO string part, not needed.